### DISCUSSION TITLE
Drop primary key constraint

### DISCUSSION THREAD

**Discussion by StNicolay** [SeaQL/sea-query#647] [Q&A]
I'm currently writing a migration that has to change a primary key. How can I drop the current one?

I'm trying to do something like this:

```mysql
ALTER TABLE users DROP PRIMARY KEY;
```

**Comment by billy1624** [comment]
Hey @StNicolay, I don't think we support such syntax at the moment. You can do it via raw SQL for now. A PR would be welcomed :)
